README.md

```diff
@@ -58,285 +58,404 @@ NOTE: You must deploy the x86 version because the Arm version produces different
         --keyring-backend test \
-        --gas auto --gas-prices 1905nhash --gas-adjustment 2 \
+        --chain-id "$CHAIN_ID" \
+        --gas auto \
+        --gas-prices 1905nhash \
+        --gas-adjustment 2 \
         --testnet \
         --yes
-        
-    provenanced tx marker withdraw gme.local 500gme.local (build/provenanced keys show -ta seller --home build/run/provenanced --keyring-backend test) \
-        --from validator \
-        --home build/run/provenanced \
+
+    $ provenanced tx marker withdraw "gme.local" "500gme.local" "$SELLER" \
+        --from "$NODE0" \
+        --home "$PIO_NODE" \
         --keyring-backend test \
-        --gas auto --gas-prices 1905nhash --gas-adjustment 2 \
+        --chain-id "$CHAIN_ID" \
+        --gas auto \
+        --gas-prices 1905nhash \
+        --gas-adjustment 2 \
         --testnet \
         --yes
     ```
 
 ### 5. Store and Instantiate the `ats-smart-contract`
 
-1. Deploy the `ats-smart-contract` to provenance. Copy the previously built `artifacts/ats-smart-contract.wasm` to the root directory of the Provenance git project
-    ```shell
-    provenanced tx wasm store ats_smart_contract.wasm \
-        --from validator \
-        --home build/run/provenanced \
+1. Copy the previously built `artifacts/ats-smart-contract.wasm` to the root directory of the Provenance git project:
+
+    ```bash
+    $ cp ats-smart-contract/artifacts/ats-smart-contract.wasm "$PIO_HOME"
+    $ cd "$PIO_HOME"
+    ```
+
+2. Deploy the `ats-smart-contract` to Provenance and store the resulting code ID:
+
+    ```bash
+    $ store_result=$(provenanced tx wasm store "ats_smart_contract.wasm" \
+        --from "$NODE0" \
+        --home "$PIO_NODE" \
+        --chain-id "$CHAIN_ID" \
         --keyring-backend test \
-        --gas auto --gas-prices 1905nhash --gas-adjustment 2 \
+        --gas auto \
+        --gas-prices 1905nhash \
+        --gas-adjustment 2 \
         --testnet \
-        --yes
+        --yes)
+    $ export CODE_ID=$(jq '.logs[0].events[] | select(.type == "store_code").attributes[] | select(.key == "code_id").value | tonumber' <<< "$store_result")
     ```
 
-2. Instantiate the contract, binding the name `ats-ex.pb` to the contract address:
-    ```shell
-    provenanced tx wasm instantiate 1 \
-   '{"name":"ats-ex", "bind_name":"ats-ex.pb", "base_denom":"gme.local", "convertible_base_denoms":[], "supported_quote_denoms":["usd.local"], "approvers":[], "executors":["'(build/provenanced keys show -ta validator --home build/run/provenanced --keyring-backend test)'"], "ask_required_attributes":[], "bid_required_attributes":[], "price_precision": "0", "size_increment": "1"}' \
-        --admin (build/provenanced keys show -ta validator --home build/run/provenanced --keyring-backend test) \
-        --label ats-ex \
-        --from validator \
-        --home build/run/provenanced \
+3. Instantiate the contract, binding the name `ats-ex.pb` to the contract address:
```

Comment:
I actually dont know if this works anymore because the `pb` base name is RESTRICTED by default now i think? which is why we create the `sc.pb` as UNRESTRICTED and then create `ats-ex.sc.pb`

Reply:
You're right. I've been using `sc.pb`. This is just a mistake on my part.